tests/components/compensation/test_sensor.py

```diff
@@ -138,34 +149,13 @@ async def test_quadratic_state(hass):
     hass.states.async_set(entity_id, 43.2, {})
     await hass.async_block_till_done()
 
-    state = hass.states.get("sensor.compensation_sensor_temperature")
+    state = hass.states.get("sensor.test")
 
     assert state is not None
 
     assert round(float(state.state), config[DOMAIN]["test"][CONF_PRECISION]) == 3.327
 
 
-async def test_numpy_errors(hass, caplog):
-    """Tests bad polyfits."""
-    config = {
-        "compensation": {
-            "test": {
-                "source": "sensor.uncompensated",
-                "data_points": [
-                    [0.0, 1.0],
-                    [0.0, 1.0],
-                ],
-            },
-        }
-    }
-    await async_setup_component(hass, DOMAIN, config)
-    await hass.async_block_till_done()
-    await hass.async_start()
-    await hass.async_block_till_done()
-
-    assert "invalid value encountered in true_divide" in caplog.text
```

Comment:
This error doesn't happen when using `Polynomial.fit`

Reply:
What happens with this data set?  What poly is returned?

Reply:
The coefficients are `[1.0, 0.0]` for this test and the polynomial always returns `1`.

Reply:
Ok, that works